cli/src/commands/developer/mod.rs

```diff
@@ -0,0 +1,159 @@
+// Copyright (C) 2019-2022 Aleo Systems Inc.
+// This file is part of the snarkOS library.
+
+// The snarkOS library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The snarkOS library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with the snarkOS library. If not, see <https://www.gnu.org/licenses/>.
+
+mod decrypt;
+pub use decrypt::*;
+
+mod deploy;
+pub use deploy::*;
+
+mod execute;
+pub use execute::*;
+
+mod scan;
+pub use scan::*;
+
+use snarkvm::{
+    file::{AleoFile, Manifest},
+    package::Package,
+    prelude::{Program, ProgramID, Transaction},
+};
+
+use anyhow::{bail, ensure, Result};
+use clap::Parser;
+use colored::Colorize;
+use std::{path::PathBuf, str::FromStr};
+
+type CurrentNetwork = snarkvm::prelude::Testnet3;
+
+/// Commands to manage Aleo accounts.
+#[derive(Debug, Parser)]
+pub enum Developer {
+    /// Decrypt a ciphertext
+    Decrypt(Decrypt),
+    /// Deploy a program.
+    Deploy(Deploy),
+    /// Execute a program function.
+    Execute(Execute),
+    /// Scan the node for records.
+    Scan(Scan),
+}
```

Comment:
We should potentially consider a simple transfer function here. Also this functionality I think can go almost verbatim into the Aleo CLI. We should talk with @collinc97 about how to properly do this. 

The following are feasible paths:
1. All of the "Developer" functions are moved into the Aleo CLI and maintained there, while the SnarkOS node management functions remain here
2. All functions within "SnarkOS" are moved to the Aleo CLI and effectively renamed/maintained there. And exist as a multi-repo.
3. The Developer functions here are duplicated and potentially have slightly different functionality in the Aleo CLI